src/containers/Roadmap/index.tsx

```diff
@@ -0,0 +1,68 @@
+import React, {FC, useEffect, useState} from 'react';
+
+import {getRoadmapByTeamName} from 'apis/roadmap';
+import {Loader} from 'components';
+import {ApiProgress} from 'constants/api-progress';
+import {RoadmapTask} from 'types/roadmap';
+import {TeamName} from 'types/teams';
+
+import {ROADMAP_TEAM_NAMES} from './constants';
+import RoadmapHero from './RoadmapHero';
+import TeamRoadmaps from './TeamRoadmaps';
+
+import './Roadmap.scss';
+
+type TeamRoadmapState = {
+  teamName: TeamName;
+  tasks: RoadmapTask[];
+}[];
+
+const Roadmap: FC = () => {
+  const [teamRoadmaps, setTeamRoadmaps] = useState<TeamRoadmapState>([]);
+  const [progressPercentage, setProgressPercentage] = useState(0);
+  const [progress, setProgress] = useState<ApiProgress>(ApiProgress.Init);
+
+  useEffect(() => {
+    setProgress(ApiProgress.Requesting);
+    let totalTasks = 0;
+    let totalCompletedTasks = 0;
+
+    const getAllRoadmaps = async () => {
+      const roadmapTasksPromises = ROADMAP_TEAM_NAMES.map(async (teamName) => {
+        const roadmapTasks = await getRoadmapByTeamName(teamName);
+        return {tasks: roadmapTasks, teamName};
+      });
+
+      return Promise.all(roadmapTasksPromises);
+    };
```

Comment:
@jamessspanggg 

What if we fetch data for all the teams at once(`/roadmaps` API) and then separate it in the front-end?
I think we can avoid multiple requests to the backend.